### PR TITLE
Add V2EX icon

### LINKED ISSUE #5836: Request: v2ex Icon
**Brand Name: v2ex**
**Website: https://www.v2ex.com/**
**Alexa rank: 2,948**
  

**Official resources for icon and color:**

### PR DESCRIPTION
**Pull request by Mayandev**
CleanShot 2022-09-02 at 23 29 22@2x

**Issue**: closes #5836
**Alexa rank**: 2,948\
**Similarweb rank**: [8,669](https://www.similarweb.com/website/v2ex.com/#overview)

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Difference preview

CleanShot 2022-09-02 at 23 32 31@2x

### CODE REVIEW

**Comment by PeterShaggyNoble** [comment]
Welcome to Simple Icons, @Mayandev, and thanks for the contribution 🙂

Can I ask which file you based your icon off? As the only place I see that icon in use is in their favicon, which wouldn't normally be enough to justify us using it, and there is no vector version available that I can see.

**Comment by Mayandev** [comment]
> Welcome to Simple Icons, @Mayandev, and thanks for the contribution 🙂
> 
> Can I ask which file you based your icon off? As the only place I see that icon in use is in their favicon, which wouldn't normally be enough to justify us using it, and there is no vector version available that I can see.

I'm sorry for the late reply, I didn't find the icon source either, this is my own design based on favoricon

**Comment by Mayandev** [comment]
> Welcome to Simple Icons, @Mayandev, and thanks for the contribution 🙂
> 
> Can I ask which file you based your icon off? As the only place I see that icon in use is in their favicon, which wouldn't normally be enough to justify us using it, and there is no vector version available that I can see.

Update

 favicon

**Comment by LitoMore** [comment]
@livid Do you have the vector version of your V2EX logo?

If not, I can help to recreate one.

**Comment by livid** [comment]
> @livid Do you have the vector version of your V2EX logo?
> 
> If not, I can help to recreate one.

Wow, this is an awesome project! Thanks a lot!

Our current favicon is a modernized redesign by @rayps based on this 16x16 pixelated legacy version created a decade ago:

https://web.archive.org/web/20060627065058/http://v2ex.com/favicon.ico

https://web.archive.org/web/20101106120836/http://www.v2ex.com/favicon.ico

Feel free to create a vector version; with your permission, we can use it as the official one.

File: icons/v2ex.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>V2EX</title><path d="M13.908 0H.904C.425 0 .038.454.038 1.01v6.98h12.196a.87.87 0 0 1 .67.337l2.83 3.482a.226.226 0 0 1 0 .275l-2.83 3.516c-.173.219-.369.393-.622.393H.038v6.998c0 .555.387 1.009.866 1.009h12.999c.287 0 .56-.135.765-.37l8.782-10.234c.33-.381.512-.903.512-1.44 0-.54-.187-1.06-.517-1.442L14.668.364a1.01 1.01 0 0 0-.76-.364z"/></svg>
```
Comment (LitoMore) [inline icons/v2ex.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>V2EX</title><path d="M14.736 1.995c.173 0 .338.072.456.198l8.526 9.094c.376.4.376 1.025 0 1.425l-8.526 9.095a.625.625 0 0 1-.456.197H.626A.625.625 0 0 1 0 21.38v-6.878h12.872a.625.625 0 0 0 .442-.182l2.024-2.025a.417.417 0 0 0 0-.59l-2.024-2.022a.625.625 0 0 0-.442-.184H0V2.62c0-.345.28-.625.625-.625z"/></svg>
```

I've recreated the logo. Let's use my new one instead.

File: _data/simple-icons.json [hunk 2]
```diff
@@ -10969,6 +10969,11 @@
                 "type": "MIT"
             }
         },
+        {
+            "title": "V2EX",
+            "hex": "222223",
```
Comment (LitoMore) [inline _data/simple-icons.json:10974]:
```suggestion
            "hex": "1F1F1F",
```

The `#1F1F1F` is picked from the darkest gradient value in `icon-192.png`.

**Comment by LitoMore** [comment]
@livid Here is the recreated one:

### Preview

CleanShot 2022-09-02 at 22 35 10@2x

### Source file

You could download the `v2ex.sketch` file from here: https://www.sketch.com/s/acc31350-dc9f-4a71-bec2-c5914debc514

Or just send the file in SVG format to you via email.

**Comment by RayPS** [comment]
> > @livid Do you have the vector version of your V2EX logo?
> > If not, I can help to recreate one.
> 
> Wow, this is an awesome project! Thanks a lot!
> 
> Our current favicon is a modernized redesign by @RayPS based on this 16x16 pixelated legacy version created a decade ago:
> 
> https://web.archive.org/web/20060627065058/http://v2ex.com/favicon.ico
> 
> https://web.archive.org/web/20101106120836/http://www.v2ex.com/favicon.ico
> 
> Feel free to create a vector version; with your permission, we can use it as the official one.

Here's the vector version of the favicon that I created for Livid.

```svg
<?xml version="1.0" encoding="UTF-8"?>
<svg width="72px" height="60px" viewBox="0 0 72 60" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
    <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
        <path d="M2,0 L43.1881209,0 C44.2949437,-3.9307556e-16 45.3522917,0.458613096 46.1086039,1.26672743 L70.4419372,27.2667274 C71.8812932,28.8046694 71.8812932,31.1953306 70.4419372,32.7332726 L46.1086039,58.7332726 C45.3522917,59.5413869 44.2949437,60 43.1881209,60 L2,60 C0.8954305,60 -8.67738547e-17,59.1045695 0,58 L0,37.5 L0,37.5 L39.3076923,37.5 L46.0895778,30.7065016 C46.479296,30.3161162 46.479296,29.6838838 46.0895778,29.2934984 L39.3076923,22.5 L39.3076923,22.5 L0,22.5 L0,2 C-3.57315355e-16,0.8954305 0.8954305,-1.91384796e-17 2,0 Z" fill="#FFFFFF"></path>
    </g>
</svg>
```

**Comment by LitoMore** [comment]
@RayPS Thanks! Will use your one instead.

I will update my sketch file with your one as well.

File: _data/simple-icons.json [hunk 3]
```diff
@@ -11555,6 +11555,11 @@
                 "type": "MIT"
             }
         },
+        {
+            "title": "V2EX",
+            "hex": "1F1F1F",
+            "source": "https://www.v2ex.com/"
```
Comment (LitoMore) [inline _data/simple-icons.json:11561]:
```suggestion
            "source": "https://www.v2ex.com"
```

File: icons/v2ex.svg [hunk 4]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>V2EX</title><path d="M14.736 1.995c.173 0 .338.072.456.198l8.526 9.094c.376.4.376 1.025 0 1.425l-8.526 9.095a.625.625 0 0 1-.456.197H.626A.625.625 0 0 1 0 21.38v-6.878h12.872a.625.625 0 0 0 .442-.182l2.024-2.025a.417.417 0 0 0 0-.59l-2.024-2.022a.625.625 0 0 0-.442-.184H0V2.62c0-.345.28-.625.625-.625z"/></svg>
```
Comment (LitoMore) [inline icons/v2ex.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>V2EX</title><path d="M.671 1.933h13.821a1.342 1.342 0 0 1 .98.425l8.166 8.725a1.342 1.342 0 0 1 0 1.834l-8.166 8.724a1.342 1.342 0 0 1-.98.426H.673A.671.671 0 0 1 0 21.395v-6.878h13.19l2.276-2.28a.336.336 0 0 0 0-.474l-2.276-2.28H0V2.604a.671.671 0 0 1 .671-.671Z"/></svg>
```

**Review by LitoMore — APPROVED** [review]
@Mayandev @PeterShaggyNoble @livid @RayPS Thank you all!

**Pull request merged**